strum_tests/tests/enum_is.rs

```diff
@@ -28,12 +28,12 @@ enum Foo {
 }
 #[test]
 fn generics_test() {
-    let foo = LifeTimeTest::One(Cow::Borrowed("Hello"));
```

Comment:
Was clippy complaining about this? It doesn't on my machine, and I prefer we avoid changing variables names based on personal preference

Reply:
I think it complains about `foo` only in `nightly`. I agree it could be ignored for now.